builds/mingw32/Makefile.mingw32

```diff
@@ -1,6 +1,6 @@
 CC=gcc
 CFLAGS=-Wall -Os -g -DDLL_EXPORT -DFD_SETSIZE=16384 -DZMQ_USE_SELECT -I.
-LIBS=-lws2_32 -lIphlpapi
+LIBS=-lws2_32 -lIphlpapi -lsodium
```

Comment:
why hardcoding libsodium? it should work with tweetnacl too and it should be an option